src/utils/discord_webhook.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+from dhooks import Embed, Webhook
+
+
+def send_discord_announcement(webhook_url: str, event: Dict):
+    w = Webhook(webhook_url)
+    embed = Embed(
+        title="📢  " + event["eventName"],
+        url=event["eventURL"],
+        description=event["eventDescription"],
+        color=0x2F3136,
+        timestamp="now",
+    )
+
+    embed.set_author(
+        name="Vijay",
+        url="https://github.com/SVijayB",
+        icon_url="https://avatars.githubusercontent.com/svijayb",
+    )
```

Comment:
This will need to be converted to OSC's GitHub profile.

Reply:
Changes to be made.
```
name => Open Source Community: VIT-AP
url => https://github.com/Open-Source-Community-VIT-AP
icon_url => https://avatars.githubusercontent.com/open-source-community-vit-ap
```

Reply:
Handled